tests/x11/reboot_gnome_pre.pm

```diff
@@ -27,8 +27,9 @@ sub run() {
 
     if (get_var("SHUTDOWN_NEEDS_AUTH")) {
         assert_screen 'reboot-auth', 15;
-        sleep 1;
+        sleep 3;
         type_password;
+        sleep 3;
```

Comment:
Sleep after typing the characters and before hitting enter? Does not make too much sense for me.
For slower typing there is https://github.com/os-autoinst/os-autoinst/pull/282 in progress

I think even that `sleep 1` after `assert_screen` is unnecessary. :)